lib/screens/sign_up.dart

```diff
@@ -102,8 +102,16 @@ class _SignUpState extends State<SignUp> {
 
         return General.instance.showSingleDialog(
           context,
-          title: Text(_localization!.trans('SIGN_UP_SUCCESS_TITLE')!),
-          content: Text(_localization!.trans('SIGN_UP_SUCCESS_CONTENT')!),
+          title: Text(
+            _localization!.trans('SIGN_UP_SUCCESS_TITLE')!,
+            style: TextStyle(
+                color: Theme.of(context).dialogTheme.titleTextStyle!.color),
+          ),
+          content: Text(
+            _localization!.trans('SIGN_UP_SUCCESS_CONTENT')!,
+            style: TextStyle(
+                color: Theme.of(context).dialogTheme.contentTextStyle!.color),
+          ),
```

Comment:
Could you explain the reason for adding the theme variable here?

Reply:
Since the `dialogBackgroundColor` theme is applied, a theme is also required for the dialog text.